rfcs/0001-bootstrapping-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: bootstrapping-rfc-process
+- Start Date: 09 February 2018
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Begin using an RFC process. Initially use a structure similar to the process used by the maintainers of Rust lang, as described in https://github.com/rust-lang/rfcs
+
+# Motivation
+[motivation]: #motivation
+
+Establish a clear, repeatable process for people to propose ideas and make decisions in a way that
+1. Allows everyone to see what decisions have been made and how they were made
+2. Allows everyone to see what proposals have been proposed and comment on them or suggest changes
+3. Works for a globally distributed group of contributors to discuss and make decisions asynchronously
+4. Can be replicated/repeated in sibling projects or spinoff projects like libp2p, IPLD, etc.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+_[Cribbed directly from [Rust-lang/rfcs README](https://github.com/rust-lang/rfcs#rust-rfcs)]_
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the IPFS community and the sub-teams.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the language and standard libraries, so that all stakeholders can be confident about the direction the language is evolving in.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The proposed RFC process is described in the [README](../README.md). It's a modified version of  the process used in the [Rust lang RFCs repository](https://github.com/rust-lang/rfcs/blob/752a02115e49c114e2d6b5247c410da69aac505c/README.md), which is dual-licensed under MIT and Apache2 licenses.
+
+Go to [README.md](../README.md) to review and comment on the details of the proposed process. _[Note: when this RFC is merged, we should update this link to point to the README.md in the specific commit that got merged.]_
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
```

Comment:
This does not feel very fleshed out yet. Few reasons I can come up with (that I don't necessary agree with, but to start something):

- Introducing too much of process, leading to developers not approaching the project in fear of too process-heavy
- It can be hard to reach consensus on some broad changes, without having **one** person being the decision maker
- Decisions can be made implicitly without going through the RFC process, and this repository still won't be the source of truth for design decisions
- Yet another repository on the IPFS Github to manage and participate in
- Not being a formal enough process to be valuable

Reply:
✅  added